plugins/autoid/plugin.js

```diff
@@ -139,9 +139,9 @@
           return newHeading;
         }
 
-        // if the text is identical (full copy), the original should retain its
+        // if the original heading text is not blank, the original should retain its
```

Comment:
I do have a few concerns about enabling paragraph ids:

1. It's not as easy to weed out potential duplication as I originally anticipated.  Doable, but compared to the general simplicity of only allowing ids on headings, noticeably more complex.

2. Ids on paragraphs potentially saved in the documents.  This could cause unexpected behavior if there is ever a future scenario where we find items by id and aren't expecting paragraphs to have ids.  We could filter them out on save by adding a rule to `htmlFilter`, but again, added rules and complexity. 

3.  I think the largest consideration is how this plugin may be used by others.  If the plugin is destined to be open source (I think this one is?  correct me if I'm wrong...), as a potential user I would not expect it to enable id's on anything other than headings.  If we want to make the id persistence more resilient, I would prefer to do it through a more robust mechanism, such as storing current heading information in a variable and doing necessary comparisons  to determine whether a particular id should persist.